modules/bmo-editing-the-hostfirmwaresettings-resource-of-a-provisioned-host.adoc

```diff
@@ -3,27 +3,26 @@
 // * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="bmo-editing-the-hostfirmwaresettings-resource_{context}"]
-= Editing the HostFirmwareSettings resource
+[id="bmo-editing-the-hostfirmwaresettings-resource-of-a-provisioned-host_{context}"]
+= Editing the HostFirmwareSettings resource of a provisioned host
 
-You can edit the `HostFirmwareSettings` of provisioned hosts.
+You can edit the `HostFirmwareSettings` for a provisioned host, delete the host from the machine set, scale down the machine set, and then scale up the machine set to make the changes take effect.
```

Comment:
Nit: This sentence is pretty long, and I lost the thread in the middle of it. Have you considered making it a list? Maybe something like:
"To make changes to the `HostFirmwareSettings` spec, you can perform any of the following actions:
* Edit the settings for a provisioned host.
* Delete the host from the machine set.
* ..."
It isn't clear to me if I should perform all of those actions to have the changes take effect or if any one of those actions will make the change take effect.
Just something to consider.

Reply:
Done. You have to perform all of them.